roles/default/kiali-remove/tasks/main.yml

```diff
@@ -164,6 +164,7 @@
   - "{{ query('k8s', namespace=kiali_vars.deployment.namespace, kind='OAuthClient', resource_name='kiali-' + kiali_vars.deployment.namespace, api_version='oauth.openshift.io/v1') if is_openshift == True else [] }}"
   - "{{ query('k8s', namespace=kiali_vars.deployment.namespace, kind='Route', resource_name='kiali', api_version='route.openshift.io/v1') if is_openshift == True else [] }}"
   - "{{ query('k8s', kind='ConsoleLink', label_selector='kiali.io/home=' + kiali_vars.deployment.namespace)}}"
+  - "{{ query('k8s', namespace=kiali_vars.deployment.namespace, kind='ConfigMap', resource_name='kiali-cabundle', api_version='v1') }}"
```

Comment:
can you add ` if is_openshift == True else []` to this? While you are here, can you also add it to line 166 - it is missing there (I wondered why I was seeing warning message while on minikube in here!! :))

Reply:
OK, both are fixed.